packages/core/faststore.config.js

```diff
@@ -45,6 +45,9 @@ module.exports = {
     shouldSplitItem: true,
   },
 
+  // Optimistic Cart
+  optimisticCart: false,
```

Comment:
It is in the right file here, but would it have any implication if we moved it to the `cart` object on line 41 and called it only `optimistic`?

Reply:
@icazevedo this information would be at the Local Storage because this is the cart we are setting as default for the user. You think should be there ? 🤔 I thought it should be separate because its a store configuration defining a behaviour and not something from the session...

Reply:
I see, makes sense.